.prettierrc

```diff
@@ -1,6 +1,11 @@
 {
   "singleQuote": true,
+
   "printWidth": 120,
+
   "trailingComma": "es5",
-  "arrowParens": "avoid"
+
+  "arrowParens": "avoid",
+
+  "endOfLine": "lf"
```

Comment:
I think this file has extra newlines in it there were added by a CRLF conversion at some point.

Reply:
Deleted src/queries/mockData.zipx

Reply:
That doesn't resolve *this* issue, where the `.prettierrc` file has extra new lines.